services/watson-extension/src/watson_extension/clients/platform/sources.py

```diff
@@ -0,0 +1,112 @@
+import abc
+import injector
+from typing import Dict, Tuple
+from aiohttp import ClientResponse
+
+from watson_extension.clients import SourcesURL
+from watson_extension.clients.identity import AbstractUserIdentityProvider
+from watson_extension.clients.platform_request import AbstractPlatformRequest
+
+
+class SourcesClient(abc.ABC):
+    @abc.abstractmethod
+    async def get_sources(self, params: Dict) -> Tuple[bool, Dict]: ...
```

Comment:
It might be good to add a type for a source

Reply:
Good suggestion 👍 
Updated to return a list of `IntegrationInfo` type objects now. We will also use this type in the integrations api routes as well.